### PR TITLE
Fixes #14994: migrate rudder_common_classes to classes_generic

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/14994

### CODE REVIEW

**Comment by amousset** [comment]
Commit modified

File: tree/20_cfe_basics/log_rudder.cf [hunk 1]
```diff
@@ -437,13 +433,8 @@ body classes rudder_common_classes(prefix)
...
 body classes rudder_common_classes_persist(prefix, persist)
 {
-    promise_kept     => { "${prefix}_kept"    , "${prefix}_ok" };
-    promise_repaired => { "${prefix}_repaired", "${prefix}_ok" };
-    repair_failed    => { "${prefix}_failed"  , "${prefix}_error" };
-    repair_denied    => { "${prefix}_denied"  , "${prefix}_error" };
-    repair_timeout   => { "${prefix}_timeout" , "${prefix}_error" };
-
-    persist_time     => ${persist};
+    inherit_from => classes_generic("${prefix}");
```
Comment (peckpeck) [inline tree/20_cfe_basics/log_rudder.cf]: why not inherit from rudder_common_classes ?

**Comment by amousset** [comment]
Commit modified

File: tree/20_cfe_basics/log_rudder.cf (hunk 1 above)
Comment (amousset) [reply tree/20_cfe_basics/log_rudder.cf]: changed, it makes more sense indeed.

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**